keyboards/fancytech/fancyalice66/info.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
This file is invalid and incorrect. Please use a JSON linter, and the Configurator to verify key position and ordering (press Ctrl+Shift+I and turn on Fast Input).

Reply:
You did not check it.
![image](https://user-images.githubusercontent.com/4781841/178453200-d78d45c1-6e73-413b-a599-359d9a4c8212.png)

Reply:
Sorry I just assumed rotation would work and didn't verify it on the configurator. How do you put the json on the configurator? I also assume the layout needs to be flat now with no rotation correct?

Reply:
I've already told you how to preview the info.json in the Configurator. And yes, it does not support rotation right now.

Reply:
Mind telling me where I can do that? Because it's not obvious.
![2022-07-12-1657617621](https://user-images.githubusercontent.com/34766434/178457109-7f7aad9b-8a17-49b0-b84a-af7edc52b864.png)

Reply:
Did you read my first comment?

Reply:
Sorry I misread, anddd it's done now.